packages/neon_framework/packages/account_repository/test/utils/http_client_builder_test.dart

```diff
@@ -11,59 +10,69 @@ class _FakeClient extends Fake implements http.Client {}
 
 class _FakeUri extends Fake implements Uri {}
 
-// ignore: subtype_of_sealed_class
-class _NeonStorageMock extends Mock implements NeonStorage {}
-
 void main() {
   final httpClient = _FakeClient();
   const userAgent = 'neon';
-  late NeonStorage storageMock;
 
   setUpAll(() {
     registerFallbackValue(_FakeUri());
-
-    storageMock = _NeonStorageMock();
-    NeonStorage.mocked(storageMock);
-    when(
-      () => storageMock.cookieStore(
-        accountID: any(named: 'accountID'),
-        serverURL: any(named: 'serverURL'),
-      ),
-    ).thenReturn(null);
   });
 
-  test('buildClient', () {
-    final credentials = Credentials((b) {
-      b
-        ..username = 'username'
-        ..appPassword = 'appPassword'
-        ..serverURL = Uri.https('serverURL');
+  group('buildClient', () {
+    test('builds client without cookie store', () {
+      final credentials = Credentials((b) {
+        b
+          ..username = 'username'
+          ..appPassword = 'appPassword'
+          ..serverURL = Uri.https('serverURL');
+      });
+
+      final client = buildClient(
+        httpClient: httpClient,
+        userAgent: userAgent,
+        credentials: credentials,
+        enableCookieStore: false,
+      );
+
+      expect(
+        client,
+        isA<NextcloudClient>()
+            .having((c) => c.authentications, 'not empty authentications', isNotEmpty)
+            .having((c) => c.baseURL, 'baseURL', equals(Uri.https('serverURL')))
+            .having(
+              (c) => c.httpClient,
+              'httpClient',
+              isA<NeonHttpClient>().having((c) => c.interceptors, 'interceptors', hasLength(3)),
```

Comment:
Please also check which interceptors are present. Only checking the count can easily go unnoticed if one is added and another on is removed.